Client/Frontend/Settings/NewTabPageTableViewController.swift

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Static
+import BraveShared
+import Shared
+
+class NewTabPageTableViewController: TableViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        navigationItem.title = Strings.NewTabPageSettingsTitle
+        tableView.accessibilityIdentifier = "NewTabPageSettings.tableView"
```

Comment:
Add empty footer `tableView.footerView = UIView(), to hide empty cells

<img width="544" alt="Zrzut ekranu 2019-11-26 o 13 56 51" src="https://user-images.githubusercontent.com/6950387/69635426-a2a15200-1054-11ea-81c9-e872ad43ed26.png">